### DISCUSSION TITLE
Read.DBC

### DISCUSSION THREAD

**Discussion by wandersonepidemiologista** [rfsaldanha/microdatasus#92] [General]
Olá! 
O Read.DBC foi retirado da CRAN e a versão arquivada não está funcionando. Há alguma previsão de atualização?
Obrigado!

**Comment by rfsaldanha** [comment]
Olá! O pacote {read.dbc} foi criado e é mantido por um outro desenvolvedor (https://github.com/danicat/read.dbc). Em tempo, vou tentar contornar o problema de instalação do pacote.

**Comment by rfsaldanha** [comment]
Olá! Disponibilizei uma nova versão do pacote ontem, que usa a versão de desenvolvimento do {read.dbc}. A princípio, o {microdatasus} deve ser instalado sem maiores problemas.

**Comment by wandersonepidemiologista** [comment]
Perfeito!
Muito obrigado!

Em ter., 30 de jan. de 2024 às 04:20, Raphael Saldanha <
***@***.***> escreveu:

**Comment by rfsaldanha** [comment]
Olá Wanderson! Você conseguiu fazer a instalação? Recebi alguns emails do GitHub com mensagens suas, mas não as achei aqui.

**Reply by wandersonepidemiologista** [reply]
Oi, Raphael! Muito obrigado pela atenção e retorno. 

Consegui sim e estou divulgando no meu blog. Dê uma olhada e compartilhe!

[www.wandersonepidemiologista.com](https://sites.google.com/epidemiologista.com/principal/tutoriais/rstudio/scripts)

**Reply by rfsaldanha** [reply]
Que bom!
